rrule/src/parser/content_line/content_line_parts.rs

```diff
@@ -0,0 +1,117 @@
+use crate::parser::{regex::get_property_name, ParseError};
+
+use super::PropertyName;
+
+#[derive(Debug, PartialEq, Clone)]
+pub(crate) struct ContentLineCaptures {
+    pub property_name: PropertyName,
+    pub parameters: Option<String>,
+    pub value: String,
```

Comment:
I think all strings here in this struct can be `&str` to avoid copying. But it's not a biggie.